centreon/src/App/MonitoringConfiguration/Domain/Aggregate/Command/Command.php

```diff
@@ -39,6 +39,10 @@ public function __construct(
         public bool $isFromMonitoringConnector,
         public ?Connector $connector,
         public ?CommandComment $comment,
+        public ?int $usedHostsCount = null,
+        public ?int $usedHostTemplatesCount = null,
+        public ?int $usedServicesCount = null,
+        public ?int $usedServiceTemplatesCount = null,
```

Comment:
```suggestion
        public ?ResourceCount $usedHostsCount = null,
        public ?ResourceCount $usedHostTemplatesCount = null,
        public ?ResourceCount $usedServicesCount = null,
        public ?ResourceCount $usedServiceTemplatesCount = null,
```

Open suggestion, can we use a value object to handle validation on the integer ? for example that they are not negative int ?